### PR TITLE
Chat Lobby

### PR DESCRIPTION
**Pull request by omidbachari**

### CODE REVIEW

File: web/static/js/socket.js [hunk 1]
```diff
@@ -53,8 +53,21 @@ let socket = new Socket("/socket", {params: {token: window.userToken}})
 
 socket.connect()
 
-// Now that you are connected, you can join channels with a topic:
-let channel = socket.channel("topic:subtopic", {})
+let channel = socket.channel("room:lobby", {})
+let chatInput = $("#chat-input")
```
Comment (cpjolicoeur) [inline web/static/js/socket.js:57]: you are using jquery selectors here but dont add jquery till the next commit

File: web/static/css/app.css [hunk 2]
```diff
@@ -1 +1,12 @@
-/* This file is for your main application css. */
\ No newline at end of file
+.chat-window {
+    border: 1px solid #e5e5e5;
+    height: 300px;
+    width: 100%;
+    font: "Comic Sans";
```
Comment (cpjolicoeur) [inline web/static/css/app.css]: FML.  Comic Sans

File: web/templates/page/index.html.eex [hunk 3]
```diff
@@ -1 +1,3 @@
 <div id="elm-target"></div>
+<div class="chat-window" id="messages"></div>
+<input class="chat-input" id="chat-input" type="text"></input>
```
Comment (cpjolicoeur) [inline web/templates/page/index.html.eex:3]: why the CSS id's that we dont use anywhere?

Comment (omidbachari) [reply web/templates/page/index.html.eex:3]: used in `socket.js`

File: web/static/css/app.css (hunk 2 above)
Comment (omidbachari) [reply web/static/css/app.css]: didn't even work

Comment (omidbachari) [reply web/static/css/app.css]: removed

File: web/static/js/socket.js (hunk 1 above)
Comment (omidbachari) [reply web/static/js/socket.js:57]: fixed

**Pull request merged**